config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -229,6 +229,7 @@ config IEEE802154_NRF5_RX_STACK_SIZE
     default 1024
 
 config OPENTHREAD_THREAD_STACK_SIZE
+    default 6144 if PSA_CRYPTO_DRIVER_CC3XX && PSA_CRYPTO_DRIVER_OBERON
```

Comment:
I know we may be in hurry, but shouldn't OpenThread provide the valid defaults for this option?

Reply:
I don't know if they have such a case when they use both drivers at the same time. We fell into this issue when we enabled support for the `ECDSA_SECP_K1_256` curve.

Reply:
So it is a very specific case.